rest_framework/permissions.py

```diff
@@ -228,8 +228,16 @@ def has_permission(self, request, view):
 
         queryset = self._queryset(view)
         perms = self.get_required_permissions(request.method, queryset.model)
+        change_perm = self.get_required_permissions('PUT', queryset.model)
+
+        user = request.user
+        if request.method == 'GET':
+            if user.has_perms(perms) or user.has_perms(change_perm):
```

Comment:
@ManishShah120 please update this block as we did in https://github.com/openwisp/openwisp-users/pull/251

Reply:
Sure, On it :+1:

Reply:
Updated the docs and simplified the code.